crates/modelardb_compression/src/models/timestamps.rs

```diff
@@ -13,22 +13,26 @@
  * limitations under the License.
  */
 
-//! Implementation of lossless compression for timestamps. Optimized compression
-//! methods are used depending on the number of data points in a compressed
-//! segment and if its timestamps have been sampled at a regular sampling
-//! interval.
+//! Implementation of MacaqueTS, a lossless compression for timestamps. MacaqueTS uses 
+//! optimized compression methods depending on the number of data points in a compressed segment
+//! and if its timestamps have been sampled at a regular sampling interval.
+//!
+//! If a segment only contains one data point, its timestamp is stored as both the segment's
```

Comment:
Maybe write the various methods timestamps can be compressed in a bullet list as is done elsewhere in the documentaiton.

Reply:
Fixed.

Reply:
Maybe make the bullet list with \* like done elsewhere in the [doc comments](https://github.com/ModelarData/ModelarDB-RS/blob/66a4abf7f10f7a790dd195f0d962fe44183100cf/crates/modelardb_server/src/remote.rs#L597).